public/index.html

```diff
@@ -24,72 +24,78 @@
     <link href="css/style.css" rel="stylesheet">
 
     <!--<script src="libs/gojs-lib/go-debug.js"></script>-->
-    <script src="libs/gojs-lib/go.js"></script>
+    <script src="libs/gojs-lib/go-debug.js"></script>
     <script src="libs/gojs-lib/Figures.js"></script>
-    <script src="libs/url-join.js"></script>
+    <script src="libs/gojs-lib/url-join.js"></script>
```

Comment:
Ne das ist falsch - die location für url-join ist: ```public/libs/url-join``` denn url join hat ja nix mit go-js zu tun. Also gehört die nicht in den ordner ```gojs-lib``` sondern nur in ```libs```